nvidia-drm-kmod: expand the comment and description

grahamperrin opened the pull request:
```text
Clarify that graphics/nvidia-drm-kmod is a meta port (not a module in itself).

Note that nvidia-drm.ko complements (non-DRM) x11/nvidia-driver.

Add context: DRM graphics subsystem.

https://bugs.freebsd.org/273311

PR: 273311

Signed-off-by: Graham Perrin <grahamperrin@gmail.com>
```

grahamperrin committed:
nvidia-drm-kmod: package description: meta

Clarify that this is a meta port (not a module in itself).

grahamperrin committed:
nvidia-drm-kmod: package description: more

Add context: Direct Rendering Manager.

Distinguish graphics/nvidia-drm-kmod from the (non-DRM) x11/nvidia-driver series.

grahamperrin committed:
nvidia-drm-kmod: package description: wrap

80 columns. I don't know what's the norm.

grahamperrin committed:
nvidia-drm-kmod/Makefile: meta (comment)

For e.g. the first line under 'Port details' at <https://www.freshports.org/graphics/nvidia-drm-kmod/> to more obviously show that this is meta. 

Compare with e.g. <https://www.freshports.org/graphics/drm-kmod/>, although let's note that Metaport (without a space) is detectably misspelt.

grahamperrin committed:
nvidia-drm-kmod: package description: update

Incorporate <https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=273311#c2> …

grahamperrin commented:
@amshafer hi

The essence of this PR – in retrospect, [`02a4caa` (#2)](https://github.com/grahamperrin/freebsd-ports/pull/2/commits/02a4caa61a7b8119b7dee46bf6ea686b56bdcd27#diff-88e498485e045af8ef7022b424bc783ca94a1b22188de29d315eca684c2e5d79L6-R6) should have been my initial commit – was **for the _meta port_ to describe itself as such**. 

Whilst not suggested in the FreeBSD Porter's Handbook, this is good practice. Please see, for example: 

- [minimalist HTML FreshPorts search results for 'meta' in the short description](https://www.freshports.org/search.php?stype=shortdescription&method=match&query=meta&num=500&orderby=port&orderbyupdown=asc&search=Search&format=html&minimal=1&branch=head) (i.e. in the `COMMENT` line). 

### More generally

There's a balance to be struck between what's expressed: 

- at home page for a port (in this case, <https://github.com/amshafer/nvidia-driver> (linked from the home icon in FreshPorts))
- in the `COMMENT` line of the `Makefile`
- in the `pkg-descr` file
- in the `pkg-message` file, the content of which is typically printed to screen after installation (see <https://docs.freebsd.org/en/books/porters-handbook/book/#porting-message> and, for example, <https://www.freshports.org/graphics/drm-510-kmod/#message>). 

In end user areas such as FreeBSD Forums and [/r/freebsd](https://www.reddit.com/r/freebsd/) I have, over the years, seen various forms of confusion surrounding both: 

1. meta port drm-kmod and the ports for related modules; and 
2. the nvidia-driver series of ports.

In my (limited) experience with the user communities, the wording of package descriptions can have a huge effect on how people – some of whom will be newcomers to a technology – understand, or misunderstand, a port. 

### Refocusing on this PR

Your <https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=273311#c2> was ideal, easily woven into [`d0de07b` (#2)](https://github.com/grahamperrin/freebsd-ports/pull/2/commits/d0de07bbc6d3e344274543020d11b81b6c1b4f65). Thanks. A raw representation of the current edition of the package message: 

- <https://github.com/grahamperrin/freebsd-ports/raw/grahamperrin-nvidia/graphics/nvidia-drm-kmod/pkg-descr>

Kick the ball around on this, if you like. 

Thanks

amshafer commented on graphics/nvidia-drm-kmod/pkg-descr:
```diff
@@ -1,9 +1,13 @@
-NVIDIA graphics hardware.
+FreeBSD port of Linux's nvidia-drm.ko Kernel module.
+
+graphics/nvidia-drm is an alternative to the x11/nvidia-driver series of ports.
```
This is just straight up wrong. nvidia-drm.ko is a *part* of the NVIDIA driver, it's an addition to `x11/nvidia-driver` not a replacement for it.

grahamperrin replied:
Thanks! I'll fix this (and the wiki). 

I'm sorry. A bad mistake. The runtime dependencies at <https://www.freshports.org/graphics/nvidia-drm-510-kmod/#requiredtorun> should have been enough for me to get this right first time.

grahamperrin replied:
[`fbc810b` (#2)](https://github.com/grahamperrin/freebsd-ports/pull/2/commits/fbc810baf718fcfde16b64c713b7e81de26d618d) … better/correct?

grahamperrin committed:
nvidia-drm-kmod: package description: major correction

Aim to resolve https://github.com/grahamperrin/freebsd-ports/pull/2#discussion_r1305662156 – correct my false assumption about this port being an alternative to the x11/nvidia-driver series.

grahamperrin commented:
Back to draft for a few minutes whilst I check one last thing …

grahamperrin committed:
nvidia-drm-kmod: package description: clarity

https://github.com/amshafer/nvidia-driver#user-content-branch-structure
notes that driver versions predating 525 lack the closed-source changes that
are necessary for use with (the port of) nvidia-drm.ko.

Attempt to convey this in the package description – for users of
x11/nvidia-driver-470 and less to be not disappointed by attempted use of
graphics/nvidia-drm-kmod.

grahamperrin commented:
If I understand correctly: nvidia-drm.ko complements the x11/nvidia-driver port alone (at the time of writing). Not, as I previously wrote, the x11/nvidia-driver _series of_ ports. 

[`aabdd12` (#2)](https://github.com/grahamperrin/freebsd-ports/pull/2/commits/aabdd12b75de153653d021a05e9675ffe02714ae) aims to cover this aspect. I'll update the wiki accordingly. 

----

@amshafer this PR should be properly ready for review now. Thanks
